scripts/tests/govcms-audit

```diff
@@ -2,8 +2,8 @@
 IFS=$'\n\t'
 set -euo pipefail
 
-APP_DIR=${APP_DIR:-/app}
-CUR_DIR=$(pwd)
+APP_DIR="${APP_DIR:-/app}"
```

Comment:
here and below - all path variables must be wrapped in quotes to avoid path splitting